wdl/GATKSVPipelineSingleSample.wdl

```diff
@@ -339,7 +338,6 @@ workflow GATKSVPipelineSingleSample {
 
     # Depth part 2
     RuntimeAttr? runtime_attr_integrate_depth_gq
-    RuntimeAttr? runtime_attr_concat_vcfs
```

Comment:
This is only used in module04, not passed to any other task.

Reply:
Yes but in 04 there should be 2 separate `runtime_attr_concat_vcfs` inputs, one for `GenotypeDepthPart2` and one for `GenotypePESRPart2`.